core/modules/filter/filter.pages.inc

```diff
@@ -432,22 +432,54 @@ function filter_format_editor_link_form($form, &$form_state, $format) {
     '#parents' => array('attributes', 'text'),
   );
 
-  $form['target'] = array(
+  $form['advanced'] = array(
+    '#title' => t('Advanced options'),
+    '#type' => 'fieldset',
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+
+  $form['advanced']['target'] = array(
     '#title' => t('Open in new window'),
     '#type' => 'checkbox',
     '#return_value' => '_blank',
     '#default_value' => isset($values['target']) ? $values['target'] : FALSE,
     '#parents' => array('attributes', 'target'),
   );
 
-  $form['class'] = array(
+  $form['advanced']['class'] = array(
     '#title' => t('CSS classes'),
     '#description' => t('Separate each additional CSS class with a space.'),
     '#type' => 'textfield',
     '#default_value' => isset($values['class']) ? $values['class'] : '',
     '#parents' => array('attributes', 'class'),
   );
 
+  $fragment_info_link = l(t('fragment'), 'https://en.wikipedia.org/wiki/Fragment_identifier', array('attributes' => array('target' => '_blank')));
+  $form['advanced']['id'] = array(
+    '#title' => t('ID'),
+    '#description' => t('Allows linking to this content using a URL !fragment. Must be unique.', array('!fragment' => $fragment_info_link)),
+    '#type' => 'textfield',
+    '#default_value' => isset($values['id']) ? $values['id'] : '',
+    '#parents' => array('attributes', 'id'),
+  );
+
+  $form['advanced']['rel'] = array(
+    '#title' => t('Relation (rel)'),
+    '#description' => t('Often used by JavaScript libraries.'),
+    '#type' => 'textfield',
+    '#default_value' => isset($values['rel']) ? $values['rel'] : '',
+    '#parents' => array('attributes', 'rel'),
+  );
+
+  $form['advanced']['title'] = array(
+    '#title' => t('Link title'),
+    '#description' => t('Populates the title attribute of the link. Usually shown as a small tooltip on hover.'),
```

Comment:
I'm not sure about 'Populates' how about... `The title attribute for the link. Usually shown as a small tooltip on hover.`

Reply:
👍